README.md

```diff
@@ -68,7 +68,7 @@ The Tenjin SDK requires the following permissions:
 <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /> <!-- Required to get network connectivity (i.e. wifi vs. mobile) -->
 ```
 
-Also, starting from **April 1st 2022**, the following permission is required to obtain Google advertising ID for apps with target API level set to 31 (Android 12). **Please add this permission as soon as possible.** You are also required to update the tenjin-android-sdk to version 1.12.8 in order to use the below permission.
+Google will eventually require apps that use `advertising_id` to set their API level to 31 (Android 12). **Please add this permission as soon as possible.** You are also required to update the tenjin-android-sdk to version 1.12.8 in order to use the below permission.
```

Comment:
Please refactor the sentence
**from**
```Google will eventually require apps that use `advertising_id` to set their API level to 31 (Android 12).```
**to**
```Google Play services will eventually require apps that use `advertising_id`(Android Advertising ID (AAID)) and have their target API level to 32 (Android 13) or above will need to declare a Google Play services AD_ID permission in their manifest file.```

**Reason**
We should not use `Google` as the above is from `Google Play services` and it is a requirement from the p`lay services ads identifier library` version 17.1.0 `com.google.android.gms:play-services-ads-identifier:17.1.0` - https://developers.google.com/android/guides/releases#september_22_2021_2

source: https://support.google.com/googleplay/android-developer/answer/6048248?hl=en

>Additionally, when apps update their target to Android 13 or above will need to declare a Google Play services normal permission in the manifest file as follows:
>
>```<uses-permission android:name="com.google.android.gms.permission.AD_ID"/>```

Reply:
Looks good you can change it like this 👍